src/openzeppelin/tests/access/test_ownable.cairo

```diff
@@ -41,65 +93,71 @@ fn test_initializer() {
 #[test]
 #[available_gas(2000000)]
 fn test_transfer_ownership() {
-    setup();
-    Ownable::transfer_ownership(OTHER());
-    assert(Ownable::owner() == OTHER(), 'Should transfer ownership');
+    let mut state = setup();
+    testing::set_caller_address(OWNER());
+    Ownable::OwnableImpl::transfer_ownership(ref state, OTHER());
+    assert(Ownable::OwnableImpl::owner(@state) == OTHER(), 'Should transfer ownership');
 }
 
 #[test]
 #[available_gas(2000000)]
 #[should_panic(expected: ('New owner is the zero address', ))]
 fn test_transfer_ownership_to_zero() {
-    setup();
-    Ownable::transfer_ownership(ZERO());
+    let mut state = setup();
+    testing::set_caller_address(OWNER());
+    Ownable::OwnableImpl::transfer_ownership(ref state, ZERO());
```

Comment:
```suggestion
    OwnableImpl::transfer_ownership(ref state, ZERO());
```

We can add `Ownable::OwnableImpl` and Ownable::OwnableCamelOnlyImpl on top of the file, and avoid the Ownable prefix in all this calls.